llvm/test/CodeGen/AMDGPU/hazards-gfx950.mir

```diff
@@ -731,17 +732,18 @@ body:             |
 ...
 
 ---
-name:            test_scalef32_sr_pk_fp4_f32_neg_opsel0_hazard
+name:            test_scalef32_sr_pk_fp4_f32_opsel0_hazard
```

Comment:
why rename here?

Reply:
This change adds `nop` for `opsel==0` as well now. So its not negative test anymore.

Reply:
It was never a negative test though, it was pre-commited for this